include/unicorn/unicorn.h

```diff
@@ -251,7 +251,7 @@ typedef enum uc_hook_type {
 #define UC_HOOK_MEM_VALID (UC_HOOK_MEM_READ + UC_HOOK_MEM_WRITE + UC_HOOK_MEM_FETCH)
 
 /*
-  Callback function for hooking memory (UC_MEM_READ, UC_MEM_WRITE & UC_MEM_FETCH)
+  Callback function for hooking memory (READ, WRITE, and FETCH)
```

Comment:
The comments were causing warnings to be emitted from const_generator.py during CI builds.